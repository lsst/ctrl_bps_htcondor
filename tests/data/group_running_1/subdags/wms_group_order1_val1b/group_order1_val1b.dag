JOB label2_val1b_val2b "label2_val1b_val2b.sub" DIR "../../jobs/label2/val1b"
JOB label2_val1b_val2a "label2_val1b_val2a.sub" DIR "../../jobs/label2/val1b"
JOB label4_val1b_val2b "label4_val1b_val2b.sub" DIR "../../jobs/label4/val1b"
JOB label4_val1b_val2a "label4_val1b_val2a.sub" DIR "../../jobs/label4/val1b"
PARENT label2_val1b_val2b CHILD label4_val1b_val2b
PARENT label2_val1b_val2a CHILD label4_val1b_val2a
DOT group_order1_val1b.dot
NODE_STATUS_FILE group_order1_val1b.node_status
SET_JOB_ATTR bps_run_quanta= "label2:2;label4:2"
SET_JOB_ATTR bps_job_summary= "label4:2;label2:2"
