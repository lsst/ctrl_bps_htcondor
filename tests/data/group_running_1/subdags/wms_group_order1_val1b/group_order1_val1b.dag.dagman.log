000 (10113.000.000) 2025-03-10 11:07:01 Job submitted from host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
    DAG Node: wms_group_order1_val1b
...
001 (10113.000.000) 2025-03-10 11:07:01 Job executing on host: <10.0.0.33:20939?addrs=10.0.0.33-20939+[2601-248-8500-b50--617a]-20939&alias=test02>
...
