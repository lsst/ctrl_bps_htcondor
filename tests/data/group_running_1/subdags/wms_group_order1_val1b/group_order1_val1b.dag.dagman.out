03/10/25 11:07:01 ******************************************************
03/10/25 11:07:01 ** condor_scheduniv_exec.10113.0 (CONDOR_DAGMAN) STARTING UP
03/10/25 11:07:01 ** /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman
03/10/25 11:07:01 ** SubsystemInfo: name=DAGMAN type=DAGMAN(9) class=CLIENT(2)
03/10/25 11:07:01 ** Configuration: subsystem:DAGMAN local:<NONE> class:CLIENT
03/10/25 11:07:01 ** $CondorVersion: 24.0.4 2025-02-10 $
03/10/25 11:07:01 ** $CondorPlatform: X86_64-AlmaLinux_9.5 $
03/10/25 11:07:01 ** PID = 18511 RealUID = 27031
03/10/25 11:07:01 ** Log last touched time unavailable (No such file or directory)
03/10/25 11:07:01 ******************************************************
03/10/25 11:07:01 Using config source: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/etc/condor/condor_config
03/10/25 11:07:01 Using local config sources: 
03/10/25 11:07:01    /etc/condor/condor_config
03/10/25 11:07:01    /etc/condor/config.d/00-minicondor
03/10/25 11:07:01    /etc/condor/config.d/00-security
03/10/25 11:07:01    /etc/condor/config.d/10-stash-plugin.conf
03/10/25 11:07:01    /etc/condor/config.d/99-lsst
03/10/25 11:07:01    /etc/condor/condor_config.local
03/10/25 11:07:01    /home/testuser/.condor/user_config
03/10/25 11:07:01 config Macros = 99, Sorted = 99, StringBytes = 3220, TablesBytes = 3652
03/10/25 11:07:01 CLASSAD_CACHING is ENABLED
03/10/25 11:07:01 Daemon Log is logging: D_ALWAYS D_ERROR D_STATUS
03/10/25 11:07:01 DaemonCore: No command port requested.
03/10/25 11:07:01 DAGMAN_USE_STRICT setting: 1
03/10/25 11:07:01 DAGMAN_VERBOSITY setting: 3
03/10/25 11:07:01 DAGMAN_DEBUG_CACHE_SIZE setting: 5242880
03/10/25 11:07:01 DAGMAN_DEBUG_CACHE_ENABLE setting: False
03/10/25 11:07:01 DAGMAN_SUBMIT_DELAY setting: 0
03/10/25 11:07:01 DAGMAN_MAX_SUBMIT_ATTEMPTS setting: 6
03/10/25 11:07:01 DAGMAN_STARTUP_CYCLE_DETECT setting: False
03/10/25 11:07:01 DAGMAN_MAX_SUBMITS_PER_INTERVAL setting: 100
03/10/25 11:07:01 DAGMAN_AGGRESSIVE_SUBMIT setting: False
03/10/25 11:07:01 DAGMAN_USER_LOG_SCAN_INTERVAL setting: 1
03/10/25 11:07:01 DAGMAN_QUEUE_UPDATE_INTERVAL setting: 30
03/10/25 11:07:01 DAGMAN_DEFAULT_PRIORITY setting: 0
03/10/25 11:07:01 DAGMAN_SUPPRESS_NOTIFICATION setting: False
03/10/25 11:07:01 allow_events (DAGMAN_ALLOW_EVENTS) setting: 114
03/10/25 11:07:01 DAGMAN_RETRY_SUBMIT_FIRST setting: True
03/10/25 11:07:01 DAGMAN_RETRY_NODE_FIRST setting: False
03/10/25 11:07:01 DAGMAN_MAX_JOBS_IDLE setting: 1000
03/10/25 11:07:01 DAGMAN_MAX_JOBS_SUBMITTED setting: 0
03/10/25 11:07:01 DAGMAN_MAX_PRE_SCRIPTS setting: 20
03/10/25 11:07:01 DAGMAN_MAX_POST_SCRIPTS setting: 20
03/10/25 11:07:01 DAGMAN_MAX_HOLD_SCRIPTS setting: 20
03/10/25 11:07:01 DAGMAN_MUNGE_NODE_NAMES setting: True
03/10/25 11:07:01 DAGMAN_PROHIBIT_MULTI_JOBS setting: False
03/10/25 11:07:01 DAGMAN_SUBMIT_DEPTH_FIRST setting: False
03/10/25 11:07:01 DAGMAN_ALWAYS_RUN_POST setting: False
03/10/25 11:07:01 DAGMAN_CONDOR_SUBMIT_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_submit
03/10/25 11:07:01 DAGMAN_CONDOR_RM_EXE setting: /work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_rm
03/10/25 11:07:01 DAGMAN_USE_DIRECT_SUBMIT setting: True
03/10/25 11:07:01 DAGMAN_PRODUCE_JOB_CREDENTIALS setting: True
03/10/25 11:07:01 DAGMAN_DEFAULT_APPEND_VARS setting: False
03/10/25 11:07:01 DAGMAN_INHERIT_ATTRS_PREFIX setting: 
03/10/25 11:07:01 DAGMAN_INHERIT_ATTRS setting: 
03/10/25 11:07:01 DAGMAN_ABORT_DUPLICATES setting: True
03/10/25 11:07:01 DAGMAN_ABORT_ON_SCARY_SUBMIT setting: True
03/10/25 11:07:01 DAGMAN_PENDING_REPORT_INTERVAL setting: 60
03/10/25 11:07:01 DAGMAN_CHECK_QUEUE_INTERVAL setting: 28800
03/10/25 11:07:01 DAGMAN_AUTO_RESCUE setting: True
03/10/25 11:07:01 DAGMAN_MAX_RESCUE_NUM setting: 100
03/10/25 11:07:01 DAGMAN_WRITE_PARTIAL_RESCUE setting: True
03/10/25 11:07:01 DAGMAN_DEFAULT_NODE_LOG setting: @(DAG_DIR)/@(DAG_FILE).nodes.log
03/10/25 11:07:01 DAGMAN_GENERATE_SUBDAG_SUBMITS setting: True
03/10/25 11:07:01 DAGMAN_MAX_JOB_HOLDS setting: 100
03/10/25 11:07:01 DAGMAN_HOLD_CLAIM_TIME setting: 20
03/10/25 11:07:01 ALL_DEBUG setting: D_FULLDEBUG
03/10/25 11:07:01 DAGMAN_SUPPRESS_JOB_LOGS setting: False
03/10/25 11:07:01 DAGMAN_REMOVE_NODE_JOBS setting: True
03/10/25 11:07:01 DAGMAN_REMOVE_JOBS_AFTER_LIMIT_CHANGE setting: False
03/10/25 11:07:01 DAGMAN will adjust edges after parsing
03/10/25 11:07:01 argv[0] == "condor_scheduniv_exec.10113.0"
03/10/25 11:07:01 argv[1] == "-Lockfile"
03/10/25 11:07:01 argv[2] == "group_order1_val1b.dag.lock"
03/10/25 11:07:01 argv[3] == "-Dag"
03/10/25 11:07:01 argv[4] == "group_order1_val1b.dag"
03/10/25 11:07:01 argv[5] == "-MaxIdle"
03/10/25 11:07:01 argv[6] == "1000"
03/10/25 11:07:01 argv[7] == "-CsdVersion"
03/10/25 11:07:01 argv[8] == "$CondorVersion: 24.0.4 2025-02-10 $"
03/10/25 11:07:01 argv[9] == "-dagman"
03/10/25 11:07:01 argv[10] == "/work/lsst_stack/w_2025_val2a/conda/envs/lsst-scipipe-10.0.0/bin/condor_dagman"
03/10/25 11:07:01 argv[11] == "-AutoRescue"
03/10/25 11:07:01 argv[12] == "1"
03/10/25 11:07:01 argv[13] == "-DoRescueFrom"
03/10/25 11:07:01 argv[14] == "0"
03/10/25 11:07:01 argv[15] == "-do_recurse"
03/10/25 11:07:01 argv[16] == "-update_submit"
03/10/25 11:07:01 Workflow batch-id: <10093.0>
03/10/25 11:07:01 Workflow batch-name: <u_testuser_DM-46294_group_fail_20250310T160455Z.dag+10093>
03/10/25 11:07:01 Workflow accounting_group: <>
03/10/25 11:07:01 Workflow accounting_group_user: <>
03/10/25 11:07:01 DAGMAN_LOG_ON_NFS_IS_ERROR setting: False
03/10/25 11:07:01 Default node log file is: </work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1b/./group_order1_val1b.dag.nodes.log>
03/10/25 11:07:01 DAG Lockfile will be written to group_order1_val1b.dag.lock
03/10/25 11:07:01 DAG Input file is group_order1_val1b.dag
03/10/25 11:07:01 Parsing 1 dagfiles
03/10/25 11:07:01 Parsing group_order1_val1b.dag ...
03/10/25 11:07:01 Adjusting edges
03/10/25 11:07:01 Dag contains 4 total nodes
03/10/25 11:07:01 Bootstrapping...
03/10/25 11:07:01 Number of pre-completed nodes: 0
03/10/25 11:07:01 MultiLogFiles: truncating log file /work/testuser/gen3work/ordering-46294/subdag/submit/u/testuser/DM-46294/group/fail/20250310T160455Z/subdags/wms_group_order1_val1b/./group_order1_val1b.dag.nodes.log
03/10/25 11:07:01 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:01 Of 4 nodes total:
03/10/25 11:07:01  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:01   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:01     0       0        0       0       2          2        0        0
03/10/25 11:07:01 0 job proc(s) currently held
03/10/25 11:07:01 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeCount = 0.0; SubmitCycleTimeSum = 0.0; ]
03/10/25 11:07:01 Registering condor_event_timer...
03/10/25 11:07:02 Submitting HTCondor Node label2_val1b_val2b job(s)...
03/10/25 11:07:02 Submitting node label2_val1b_val2b from file label2_val1b_val2b.sub using direct job submission
03/10/25 11:07:02 	assigned HTCondor ID (10115.0.0)
03/10/25 11:07:02 Submitting HTCondor Node label2_val1b_val2a job(s)...
03/10/25 11:07:02 Submitting node label2_val1b_val2a from file label2_val1b_val2a.sub using direct job submission
03/10/25 11:07:03 	assigned HTCondor ID (10116.0.0)
03/10/25 11:07:03 Just submitted 2 jobs this cycle...
03/10/25 11:07:03 DAG status: 0 (DAG_STATUS_OK)
03/10/25 11:07:03 Of 4 nodes total:
03/10/25 11:07:03  Done     Pre   Queued    Post   Ready   Un-Ready   Failed   Futile
03/10/25 11:07:03   ===     ===      ===     ===     ===        ===      ===      ===
03/10/25 11:07:03     0       0        2       0       0          2        0        0
03/10/25 11:07:03 0 job proc(s) currently held
03/10/25 11:07:03 DAGMan Runtime Statistics: [EventCycleTimeCount = 0.0; EventCycleTimeSum = 0.0; LogProcessCycleTimeCount = 0.0; LogProcessCycleTimeSum = 0.0; SleepCycleTimeCount = 0.0; SleepCycleTimeSum = 0.0; SubmitCycleTimeAvg = 0.1422841548919678; SubmitCycleTimeCount = 1.0; SubmitCycleTimeMax = 0.1422841548919678; SubmitCycleTimeMin = 0.1422841548919678; SubmitCycleTimeStd = 0.1422841548919678; SubmitCycleTimeSum = 0.1422841548919678; ]
03/10/25 11:07:04 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:04 Reassigning the id of job label2_val1b_val2b from (10115.0.0) to (10115.0.0)
03/10/25 11:07:04 Event: ULOG_SUBMIT for HTCondor Node label2_val1b_val2b (10115.0.0) {03/10/25 11:07:02}
03/10/25 11:07:04 Number of idle job procs: 1
03/10/25 11:07:04 Reassigning the id of job label2_val1b_val2a from (10116.0.0) to (10116.0.0)
03/10/25 11:07:04 Event: ULOG_SUBMIT for HTCondor Node label2_val1b_val2a (10116.0.0) {03/10/25 11:07:02}
03/10/25 11:07:04 Number of idle job procs: 2
03/10/25 11:07:08 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:08 Event: ULOG_EXECUTE for HTCondor Node label2_val1b_val2b (10115.0.0) {03/10/25 11:07:07}
03/10/25 11:07:08 Number of idle job procs: 1
03/10/25 11:07:11 Currently monitoring 1 HTCondor log file(s)
03/10/25 11:07:11 Event: ULOG_EXECUTE for HTCondor Node label2_val1b_val2a (10116.0.0) {03/10/25 11:07:10}
03/10/25 11:07:11 Number of idle job procs: 0
